### PR TITLE
r1.1

### PR DESCRIPTION
**Pull request by hdamker**
#### What type of PR is this?

* documentation
* correction of info object in YAML

#### What this PR does / why we need it:

- Expanded "SED" into the API name
- Corrected API Version number and release number in API Checklist
- Replaced absolute links into main with relative links within the Checklist (they will always point to the right version within a release/tag)
- Corrections within info object in YAML:
  - Removed contact field
  - Brought license and x-camara-commonalities fields on top level within info object.

#### Which issue(s) this PR fixes:

none (minor PR)

### CODE REVIEW

**Review by Kevsy — APPROVED** [review]
LGTM

File: documentation/API_documentation/simple-edge-discovery_API_Readiness_Checklist.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-|  6 | User stories                                 |   O   |         O         |    O    |    M   |  Y   | [link](https://github.com/camaraproject/SimpleEdgeDiscovery/tree/main/documentation/API_documentation/SED_User_Story.md) |
-|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |  Y   | [link](https://github.com/camaraproject/SimpleEdgeDiscovery/tree/main/code/Test_definitions) |
-|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |  N   |  |
+|  5 | API documentation                            |   M   |         M         |    M    |    M   |  Y   | [link](/code/API_definitions/simple-edge-discovery.yaml) |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |  Y   | [link](/documentation/API_documentation/SED_User_Story.md) |
+|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |  Y   | [link](/code/Test_definitions) |
```
Comment (tanjadegroot) [inline documentation/API_documentation/simple-edge-discovery_API_Readiness_Checklist.md:13]: The feature still refers to version 0.10.0. 
Question: is it necessary to reference the API version ? as it anyway is in the same release package ?

File: documentation/API_documentation/SimpleEdgeDiscovery_User_Story.md
[the remark this answers is not in the document]
Comment (Kevsy) [reply documentation/API_documentation/SimpleEdgeDiscovery_User_Story.md:1]: Thanks, fixed in new commit

File: code/API_definitions/simple-edge-discovery.yaml [hunk 2]
```diff
@@ -219,12 +219,12 @@ info:
...
 
   ---
- contact:
-  email: sp-edc@lists.camaraproject.org
+
  license:
   name: Apache 2.0
   url: https://www.apache.org/licenses/LICENSE-2.0.html
-  x-camara-commonalities: 0.4.0
+ x-camara-commonalities: 0.4.0
```
[the remark this answers is not in the document]
Comment (Kevsy) [reply code/API_definitions/simple-edge-discovery.yaml]: Thanks, fixed in new commit

File: documentation/API_documentation/simple-edge-discovery_API_Readiness_Checklist.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
-|  5 | API documentation                            |   M   |         M         |    M    |    M   |  Y   | [link](https://github.com/camaraproject/SimpleEdgeDiscovery/blob/main/code/API_definitions/simple-edge-discovery.yaml) |
-|  6 | User stories                                 |   O   |         O         |    O    |    M   |  Y   | [link](https://github.com/camaraproject/SimpleEdgeDiscovery/tree/main/documentation/API_documentation/SED_User_Story.md) |
-|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |  Y   | [link](https://github.com/camaraproject/SimpleEdgeDiscovery/tree/main/code/Test_definitions) |
-|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |  N   |  |
+|  5 | API documentation                            |   M   |         M         |    M    |    M   |  Y   | [link](/code/API_definitions/simple-edge-discovery.yaml) |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |  Y   | [link](/documentation/API_documentation/SED_User_Story.md) |
```
[the remark this answers is not in the document]
Comment (Kevsy) [reply documentation/API_documentation/simple-edge-discovery_API_Readiness_Checklist.md]: Fixed in new commit, thanks

File: documentation/API_documentation/simple-edge-discovery_API_Readiness_Checklist.md (hunk 1 above)
Comment (Kevsy) [reply documentation/API_documentation/simple-edge-discovery_API_Readiness_Checklist.md:13]: Good point, I've removed it.

**Review by Kevsy — APPROVED** [review]
Tanja's feedback now incorporated - LGTM.

**Comment by tanjadegroot** [comment]
LGTM - From release mgmt perspective all is OK

please update the API release tracker (and remove there the r1.1 in the public release field :-) ) almost there.

**Review by javierlozallu — APPROVED** [review]

**Pull request merged**